tests/PhpFpmTest.php

```diff
@@ -131,14 +150,16 @@ public function test_use_version_if_already_linked_php_will_unlink_before_instal
         $brewMock->shouldReceive('getAllRunningServices')->andReturn(collect());
         $brewMock->shouldReceive('stopService');
 
+        $nginxMock->shouldReceive('restart');
+
         // Test both non prefixed and prefixed
         $this->assertSame('php@7.2', $phpFpmMock->useVersion('php@7.2'));
     }
 }
 
 class StubForUpdatingFpmConfigFiles extends PhpFpm
 {
-    public function fpmConfigPath()
+    public function fpmConfigPath($phpVersion = null)
     {
         return __DIR__.'/output/fpm.conf';
     }
```

Comment:
I'd love to see new tests for the new methods that were introduced in this PR. Let me know if you need help from me making this happen.

Reply:
I'll try to contribute some tests on this PR. 👍